src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationInvalidSwitchIndent.java

```diff
@@ -101,6 +101,10 @@ private void method1() { //indent:4 exp:4
 	    		break; //indent:16 exp:20 warn
 	    	else //indent:12 exp:16 warn
 	    		break; //indent:16 exp:20 warn
+            case 4: //indent:12 exp:12
```

Comment:
I had to modify this input file for test and mutation coverage due to changes in `abstractExpressionHandler.java`

Reply:
it is always ok to extend tests.